src/vaults/VaultRouter.sol

```diff
@@ -63,19 +63,19 @@ contract VaultRouter is Auth, Multicall, Recoverable, IVaultRouter {
         super.multicall(data);
 
         if (!wasBatching) {
-            gateway.endBatching();
+            gateway.endBatching{value: msg.value}(msg.sender);
```

Comment:
Note for VaultRouter actions:
- If no batching, the messages are subsidized at the vault level.
- If batching, the messages are paid in the multicall.